File: Sopra/finished_project/sopra-fs19-group03-server/src/main/java/ch/uzh/ifi/seal/soprafs19/dto/incoming/AuthorizationCredentials.java
package ch.uzh.ifi.seal.soprafs19.dto.incoming;

import java.io.Serializable;

public class AuthorizationCredentials implements Serializable, Message {
    private String token;
    private Long id;

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }
}

File: Sopra/finished_project/sopra-fs19-group03-client/src/components/shared/models/outgoingMessages/AuthorizationCredentials.js

class AuthorizationCredentials{
    constructor(token, id){
        this.token = token;
        this.id = id;
    }

    getToken(){ return this.token;}
    setToken(token){this.token = token;}
    getId(){return this.id;}
    setId(id){this.id = id;}
}
export default AuthorizationCredentials;